p2p/host/basic/basic_host.go

```diff
@@ -654,6 +652,8 @@ func (h *BasicHost) NewStream(ctx context.Context, p peer.ID, pids ...protocol.I
 		}, nil
 	}
 
+	pidStrings := protocol.ConvertToStrings(pids)
```

Comment:
This won't be needed any more once you pull in https://github.com/multiformats/go-multistream/pull/95, right?

Reply:
yes. This would be resolved.

Reply:
this is fixed.